src/selectors/index.ts

```diff
@@ -0,0 +1,12 @@
+import { createSelector } from 'reselect';
+import { Comment } from '../state/comments'
+import { State } from '../state'
```

Comment:
**Nitpick:** I suggest using `import type` to import static types. https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export

But you need to be on at least 3.8. This will avoid potential circular imports. Generally a good practice.

Reply:
Done, thanks